authcontroller.py

```diff
@@ -548,14 +554,14 @@ def confirm_organisation(token=None):
 
     # For now only GET method is implemented will need post method for organisation
     # to enter client secret and client key for orcid
+    organisation = Organisation.get(email=email)
```

Comment:
What happens if the organisation record after having sent the invitation got deleted?